Assets/HoloToolkit/SpectatorView/Scripts/Utils/TweenAlpha.cs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.using UnityEngine;
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace HoloToolkit.Unity.SpectatorView
+{
+    /// <summary>
+    /// Utility for fading out marker
+    /// </summary>
+    public class TweenAlpha : MonoBehaviour
```

Comment:
Seems like this could be a common utility that doesn't need to live in this feature folder.